source/_patterns/04-pages/research-article/research-article.yaml

```diff
@@ -40,10 +40,16 @@ body:
           angiosperm origins.
   - template: atoms-paragraph
     arguments:
-      text: |
-        From oranges to apples, flowering plants produce most of the fruits and vegetables that we can see on display in
-        a supermarket. While we may take little notice of the poppy fields and plum blossoms around us, how flowers came
-        to be has been an intensely debated mystery.
+      text:
+        - |
+            From
+        - template: atoms-sub
+          arguments:
+            text: ' oranges '
```

Comment:
Would it be more consistent with a real example if the whitespace was not included here but in the previous and subsequent stings?

Will this produce the following?:

```
<p>From<sub> oranges </sub>to apples, flowering plants produce most of the fruits...</p>
```

Reply:
It would. Did this to start with to avoid having to wrap the longer strings in quotes to avoid the whitespace stripping, but I've just found out about YAML multiline whitespace handling, and we can do https://github.com/libero/pattern-library/blob/be651e14ff1f64a86c78fa8e971b98eacf88a9e0/source/_patterns/04-pages/research-article/research-article.yaml#L44-L51